colin/core/target.py

```diff
@@ -9,10 +14,37 @@ class Target(object):
 
     def __init__(self, name):
         self.name = name
+        self.instance = Target._get_conu_instance(name)
+
+    @staticmethod
+    def _get_conu_instance(target_name):
+        with DockerBackend() as backend:
+
+            try:
+                cont = backend.ContainerClass(image=None,
+                                              container_id=target_name)
+                return cont
+            except NotFound:
+                name_split = target_name.split(':')
```

Comment:
Probably the most comprehensive logic to parse image names is here: https://github.com/projectatomic/atomic-reactor/blob/master/atomic_reactor/util.py#L57

the current code should be okay for now, just food for thought for future

Reply:
I was thinking about that since it is not the first time I am writing this piece of code...;-)

I'll make a new "starter" issue for that.